third_party/silabs/silabs_board.gni

```diff
@@ -62,7 +62,7 @@ if (silabs_board == "") {
 assert(silabs_board != "", "silabs_board must be specified")
 
 # Si917 WIFI board ----------
-if (silabs_board == "BRD4338A") {
+if (silabs_board == "BRD4338A" || silabs_board == "BRD2605A") {
```

Comment:
do we need changes in scripts/build (targets.py, efr32.py and unit test txt files)?

Reply:
@andy31415 - Added the brd2605a board in efr32.py, targets.py and examples-efr32.yaml